contracts/StableAssetApplication.sol

```diff
@@ -10,6 +10,14 @@ import "@openzeppelin/contracts-upgradeable/utils/math/SafeMathUpgradeable.sol";
 import "./interfaces/IWETH.sol";
 import "./StableAsset.sol";
 
+interface Ipool {
```

Comment:
maybe not create an interface here but a separate one in `interfaces`?

Reply:
done